src/commands/load.js

```diff
@@ -0,0 +1,33 @@
+let path = require('path');
+
+module.exports = () => {
+	return {
+		run: function(args, api, event) {
```

Comment:
Mix of ' ' and '\t' spacing. For consistency with the main Concierge project and the rest of this repo, 4 space indentation should be used.

(On a personal note: I don't really care which indentation type is used tbh, as long as it is consistent)